tests/testthat/test-load.r

```diff
@@ -0,0 +1,10 @@
+context("Loading")
+
+test_that("Package root is working directory when loading", {
+  expect_message(load_all("testLoadDir"), "[|].*/testLoadDir[|]")
+})
+
+test_that("Loading a package subdirectory throws an error", {
```

Comment:
@hadley What is the point of calling `create_description()` in `load_all()` if the `DESCRIPTION` file is missing? What was the original intention? The `load_all()` fails anyway if `R` and/or `man` is missing.